client/app/components/ApplicationArea/ApplicationLayout/DesktopNavbar.jsx

```diff
@@ -71,9 +72,9 @@ export default function DesktopNavbar() {
   const canCreateAlert = currentUser.hasPermission("list_alerts");
 
   return (
-    <div className="desktop-navbar">
+    <nav className="desktop-navbar" role="navigation">
```

Comment:
`nav` shouldn't need an additional role.

Reply:
It is actually recommended by WAI ARIA, as navs are not supported in older browsers. Of course this is questionable, since these are not under our supported browsers. I'll just remove it

Reply:
Good news is that jsx/a11y actually enforces this behavior (try adding a role="link" to an anchor 🤠)